Sources/Hummingbird/Codable/JSON/JSONCoding.swift

```diff
@@ -48,5 +48,10 @@ extension JSONDecoder: HBRequestDecoder {
 /// `HBRequestDecoder` and `HBResponseEncoder` both require conformance to `Sendable`. Given
 /// `JSONEncoder`` and `JSONDecoder`` conform to Sendable in macOS 13+ I think I can just
 /// back date the conformance to all versions of Swift, macOS we support
-extension JSONDecoder: @unchecked Sendable {}
+#if $RetroactiveAttribute
```

Comment:
```suggestion
#if hasFeature(RetroactiveAttribute)
```
The issue causing `hasFeature` to not work correctly has been resolved on main (not on 5.10, but `RetroactiveAttribute` is not available there anyway.